### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+project: mdformat
```
Comment (xansec) [inline Mayhemfile:1]: Tests has been changed to testsuite. Please make this change in this file as well. You may also omit it entirely.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Mayhemfile (hunk 1 above)
Comment (ebell495) [reply Mayhemfile:1]: I've fixed this

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**